File: javascript/src/models/googleCloudPubSubConfigOut.ts
// this file is @generated

export interface GoogleCloudPubSubConfigOut {
  projectId: string;
  topicId: string;
}

export const GoogleCloudPubSubConfigOutSerializer = {
  _fromJsonObject(object: any): GoogleCloudPubSubConfigOut {
    return {
      projectId: object["projectId"],
      topicId: object["topicId"],
    };
  },

  _toJsonObject(self: GoogleCloudPubSubConfigOut): any {
    return {
      projectId: self.projectId,
      topicId: self.topicId,
    };
  },
};

File: rust/src/models/google_cloud_pub_sub_config_out.rs
// this file is @generated
use serde::{Deserialize, Serialize};

#[derive(Clone, Debug, PartialEq, Deserialize, Serialize)]
pub struct GoogleCloudPubSubConfigOut {
    #[serde(rename = "projectId")]
    pub project_id: String,

    #[serde(rename = "topicId")]
    pub topic_id: String,
}
